GDAXSharp/Network/HttpRequest/HttpRequestMessageService.cs

```diff
@@ -1,39 +1,32 @@
-﻿using System;
+﻿using GDAXSharp.Network.Authentication;
+using GDAXSharp.Shared.Utilities.Clock;
+using GDAXSharp.Shared.Utilities.Extensions;
+using System;
 using System.Globalization;
 using System.Net.Http;
-using System.Security.Cryptography;
 using System.Text;
-using GDAXSharp.Network.Authentication;
-using GDAXSharp.Shared.Utilities.Clock;
-using GDAXSharp.Shared.Utilities.Extensions;
 
 namespace GDAXSharp.Network.HttpRequest
 {
-    public class HttpRequestMessageService : IHttpRequestMessageService
+    public class HttpRequestMessageService : AbstractRequest, IHttpRequestMessageService
     {
         private const string ApiUri = "https://api.gdax.com";
 
         private const string SandBoxApiUri = "https://api-public.sandbox.gdax.com";
 
-        private readonly IAuthenticator authenticator;
-
-        private readonly IClock clock;
-
-        private readonly bool sandBox;
-
-        public HttpRequestMessageService(IAuthenticator authenticator, IClock clock, bool sandBox)
+        public HttpRequestMessageService(
+            IAuthenticator authenticator,
+            IClock clock,
+            bool sandBox) : base(authenticator, clock, sandBox)
```

Comment:
hate to be a pain about styling but please new line this:

````
public HttpRequestMessageService(
           IAuthenticator authenticator,
           IClock clock,
           bool sandBox) 
                  : base(authenticator, clock, sandBox)
````

just like to keep the codebase as consistent and clean as possible